tools/RAiDER/models/weatherModel.py

```diff
@@ -485,9 +485,23 @@ def checkValidBounds(
         ll_bounds : np.ndarray
 
         Returns:
-        bool    True if the ll_bounds represent a valid area for the weather model
+        bool    The weather model object
         '''
-        return box(ll_bounds[2], ll_bounds[0], ll_bounds[3], ll_bounds[1]).intersects(self._valid_bounds)
+        S, N, W, E = ll_bounds
```

Comment:
This looks great; only I would override the function in the hrrr module. So this function stays the same, but you define a new method with the same name in the `HRRR` weather model class with everything under line 494.